youtube_dl/extractor/common.py

```diff
@@ -1053,6 +1053,15 @@ def extract_automatic_captions(self, *args, **kwargs):
     def _get_automatic_captions(self, *args, **kwargs):
         raise NotImplementedError("This method must be implemented by subclasses")
 
+    def _multiple_opener_supported(self):
+        return hasattr(self._downloader, 'use_opener')
```

Comment:
This can probably be removed, nobody is going to use their own class for `_downloader`.